examples/events/src/main.rs

```diff
@@ -55,8 +55,9 @@ impl Application for Events {
                 Command::none()
             }
             Message::EventOccurred(event) => {
-                if let Event::Window(window::Event::CloseRequested) = event {
-                    window::close()
+                if let Event::Window(id, window::Event::CloseRequested) = event
+                {
+                    window::close(id)
```

Comment:
This is currently the only way to close a window if a user manually closes it with whatever OS-specific close button there is; listen to the `CloseRequested` event, and then manually return the `window::close` command in application update. Not sure if we want a more "opt-in" way to respond to `CloseRequested` events. Maybe closing the window automatically should be the default? Could repurpose the `exit_on_close_requested` setting to indicate this behavior maybe.

Reply:
Changed to make more sense in 83c7870c569a2976923ee6243a19813094d44673